### ISSUE TITLE
Device : Flexi

### ISSUE THREAD

**Issue by hive** [hive/com.groups#32]
Create a new 'flexi' device which will allow devices to be added to a group which may not contain all of the capabilities. 

This would allow you to group completely different devices and control them from the one device. A simple, example of this would be to have a light group (on/off, dim, colour, hue, power meter) - with all types of lights with in it. When changing colours only the devices which an would do so. 

A more extreme example might be create a a heater group which contains all of the thermostats, as well as a heater and a HVAC.

**Comment by Pachoracing** [comment]
Yes I would love something like this, like powering on all the different home theater equipment and stuff like that
